docs/migration.md

```diff
@@ -0,0 +1,167 @@
+
+
+Migration from v1.2.8 to v2.0.0
+-------------------------------
+
+  
+
+This document is a guide to help you migrate from rdiff-backup v1.2.8
+(legacy version) to rdiff-backup v2.0.0 (new version).
+
+  
+
+This new version of rdiff-backup is compatible with repositories created
+with the legacy version. In other words, the data on the disk is fully
+compatible between the two. But the network protocol is not. For this

```

Comment:
The sentence `But the network protocol is not.` should be a full sentence and in bold, or the first sentence in the next paragraph to ensure all readers notice it in case they are "scanning" the text and not reading full paragraphs. This is perhaps the most important sentence in the whole document.